### PR TITLE
Feature: Vite Build

### PR DESCRIPTION
**Pull request by VitroidFPV**
`yarn build` outputs into `src/dist/`. Should be ready to be deployed

### CODE REVIEW

**Comment by VitroidFPV** [comment]
> for anyone else reviewing. re-execution of `yarn install` required.
> 
> @VitroidFPV , `yarn dev` and/or `yarn vite` failing for me. image
> 
> However `yarn preview` and `yarn review` successful and blazingly fast after the build.

It seems to be an issue with the copy plugin, I'm not sure why it's going wrong when it should just be copying some files. Will investigate

**Comment by nerdCopter** [comment]
thanks!
these error do not exist in NWjs version.
image

**Comment by VitroidFPV** [comment]
@nerdCopter the plugin used copies the files for builds but serves them directly when used in dev. I picked a different one that will just copy them no questions asked in both envs. Try now!

**Comment by VitroidFPV** [comment]
I will look into the LESS loading issues, but that may be for another PR

**Comment by nerdCopter** [comment]
same exact result.
`yarn gulp debug` is good.
`yarn build ; yarn preview` is good.
`yarn dev` is broken.

**Comment by VitroidFPV** [comment]
> same exact result. `yarn gulp debug` is good. `yarn build ; yarn preview` is good. `yarn dev` is broken.

Did you re-install the packages again? Can confirm it working locally now on :8000
image

**Comment by nerdCopter** [comment]
`yarn install --check-files`
but same results. i'm not sure why :man_shrugging:  i cleared my cache, disabled blockers, & restarted browser as well.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> FAIL
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Works for me:

```bash
    git clean -xdf
    yarn && yarn dev
```

image

**Review by nerdCopter — APPROVED** [review]
approving. 
  working today. did not have to do anything different. 🔮🤷‍♂️

File: vite.config.js
Comment (chmelevskij) [inline vite.config.js:63]: Hmm, guessing we can't have `/dist` in the `root` because of `root` setting in vite config 🤔 Having it in `src` feels like will backfire

Comment (nerdCopter) [reply vite.config.js:63]: interesting and good catch. but it is ignored by default (`.gitignore `)

Comment (VitroidFPV) [reply vite.config.js:63]: It was already set up that way in #3292. But yeah in `src` it will probably come back to bite us. What do you suggest?

Comment (haslinghuis) [reply vite.config.js:63]: Should we look into

image

Comment (nerdCopter) [reply vite.config.js:63]: > Should we look into

i don't really understand the errors. they do go away upon re-execution. they tend to only present themselves with new code/execution

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
